tools/distrib/clang_tidy_code.sh

```diff
@@ -21,6 +21,22 @@ set -ex
 cd $(dirname $0)/../..
 REPO_ROOT=$(pwd)
 
+# grep targets with manual tag, which is not included in a result of bazel build using ...
+# let's get a list of them using query command and pass it to gen_compilation_database.py
+export MANUAL_TARGETS=$(bazel query 'attr("tags", "manual", tests(//test/cpp/...))' | grep -v _on_ios)
+
+# generate a clang compilation database
+tools/distrib/gen_compilation_database.py \
+  --include_headers \
+  --ignore_system_headers \
+  --dedup_targets \
+  "//:*" \
```

Comment:
I wish there was a better way of identifying the targets we want to include in compdb (I'd like something that's zero-maintenance), but I'm assuming you already explored that option and could not come up with anything better (but still simple).
Perhaps add a TODO that mentions what would ideally be the set of targets we'd like to list here (is it "all C/C++ code in this repo"?), and that the current list of targets is the best approximation of that we could get.

Reply:
Yeah, it's not easy and probably not possible to get it fully automatic. I updated the comment slightly to explain what's done here.